<launch>
   <include file="$(find ros_pololu_servo)/launch/pololu_example.launch"/>

  <!-- Paramter Configuration for the race only-->
   <node pkg="state_machine" type="left_master_state_machine.py" name="master_state_machine" output="screen">
     <param name="stop_motor" type="double" value="0.0" />
     <param name="slow_motor" type="double" value="-0.22" />
     <param name="servo_zero" type="double" value="0.115" />
     <param name="high_speed" type="double" value="-0.63" />
     <param name="turn_depth" type="double" value="5500" />
     <param name="reverse_motor" type="double" value="0.85" />
   </node>

  <!-- Paramter configuration for depth information-->
   <node pkg="camera" type ="depth" name="depth_stream" output="screen">
     <param name="frame_rate" type="int" value="60" />
     <param name="resolution_height" type="int" value="480" />
     <param name="resolution_width" type="int" value="640" />
   </node>

    <node pkg="pid_controller" type ="left_output_publisher.py" name="left_output_publisher" output="screen"/>
    
<!--    <node pkg="pid_controller" type ="kp_publisher.py" name="kp_publisher" output="screen"/> -->

</launch>
